src/templates/snek/gql/tasks/user.ts

```diff
@@ -0,0 +1,149 @@
+//#region > Imports
+//> Parent Task
+// Contains the SNEK parent task
+import SnekTasks from "./index";
+//> Types
+// Contains the type declaration for apollo results
+import { ApolloResult } from "./index";
+//#endregion
+
+//#region > Interfaces
+/**
+ * @interface RegistrationData defines the structure of the registration result
+ *                             data.
+ */
+interface RegistrationData {
+  result: string;
+}
+
+/**
+ * @interface CacheData defines the structure of the cache result
+ *                      data.
+ */
+interface CacheData {
+  user: {
+    id: number;
+  };
+}
+
+/**
+ * @interface ProfileData defines the structure of the profile result
+ *                        data.
+ */
+interface ProfileData {
+  username: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  verified: string;
+  platformData: string;
+  sources: string;
+  bids: string;
+  tids: string;
+}
+
+/**
+ * @interface WhoamiData defines the structure of the whoami result
+ *                        data.
+ */
+interface WhoamiData {
+  whoami: {
+    username: string;
+  };
+}
+//#endregion
+
+//#region > Classes
+/** @class A set of session aware tasks */
+class SnekGqlUserTasks {
+  /**
+   * @constructor
+   * @author Nico Schett <contact@schett.net>
+   * @param {string} parent The parent task
+   */
```

Comment:
Should have some sort of comment.

?

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3